src/components/Confirm/Confirm.js

```diff
@@ -1,7 +1,10 @@
 import React, {Component, PropTypes} from 'react';
 import Promise from 'bluebird';
 
-import {Modal, ModalContent, ModalFooter, ModalHeader} from 'stardust';
+import Modal from 'src/components/Modal/Modal';
+import ModalContent from 'src/components/Modal/ModalContent';
+import ModalFooter from 'src/components/Modal/ModalFooter';
+import ModalHeader from 'src/components/Modal/ModalHeader';
```

Comment:
Importing from `stardust` was causing a failure probably due to some circular import issue.  When in the `src` directory, let's only import exact paths.

The issue was, `stardust.Confirm` was undefined when it should have been a `function`.